umm-lib/src/cmr/umm/echo10/collection/campaign.clj

```diff
@@ -36,16 +31,3 @@
                      (x/element :ShortName {} short-name)
                      (when long-name
                        (x/element :LongName {} long-name))))))))
-
-
-(comment
```

Comment:
Because this is umm-lib collection and this code is not used.  This namespace needs to be deleted, but I can't deleted because I have to change the tests and update ACL and user stuff from moch echo. But this is dead code and so we don't need the comment anymore.

Reply:
True, if that was the reason -- comments will no longer throw linting errors after my linting PR with the config file merges.

Reply:
no, it should stay out if you don't think it helps.